code/campaignclassic/src/main/java/com/adobe/marketing/mobile/AEPMessagingService.java

```diff
@@ -105,9 +115,15 @@ public static boolean handleRemoteMessageData(
         AEPPushPayload payload;
```

Comment:
Does this need to be `public`? If so, is it to accommodate only the testing effort internally, or do we anticipate its use by customers?

Reply:
i've made it package private as it is used by the extension only